tests/Annotations/Provider/QueueCommandsProviderTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace tests\eLife\Annotations\Provider;
+
+use Aws\Sqs\SqsClient;
+use eLife\Annotations\Provider\QueueCommandsProvider;
+use eLife\ApiClient\HttpClient;
+use eLife\ApiSdk\ApiSdk;
+use eLife\Bus\Limit\Limit;
+use eLife\Bus\Queue\QueueItemTransformer;
+use eLife\Bus\Queue\WatchableQueue;
+use eLife\HypothesisClient\ApiSdk as HypothesisApiSdk;
+use eLife\HypothesisClient\HttpClient\HttpClient as HypothesisHttpClient;
+use eLife\Logging\Monitoring;
+use Knp\Console\Application as ConsoleApplication;
+use Knp\Provider\ConsoleServiceProvider;
+use LogicException;
+use PHPUnit_Framework_TestCase;
+use Psr\Log\LoggerInterface;
+use Silex\Application;
+
+/**
+ * @covers \eLife\Annotations\Provider\QueueCommandsProvider
+ */
+final class QueueCommandsProviderTest extends PHPUnit_Framework_TestCase
```

Comment:
I don't think this test is very predictive of bugs, since Providers are factories and stubbing all of its dependencies takes away all their responsibility, to put together objects. It could have more value if it used the real provider registered in the Kernel. Otherwise it could be dropped for ease of maintenance, and all the construction logic would be tested at an higher level by the build (which will run commands for example).